src/Persistence/Initialization/VersionSeasonSix/ChaosMixes.cs

```diff
@@ -82,18 +82,20 @@ public override void Initialize()
         cherryBlossomSpirit.ItemCraftings.Add(this.CherryBlossomEventCrafting());
     }
 
-    private ItemCrafting ItemLevelUpgradeCrafting(byte craftingNumber, byte targetLevel, int money)
+    private ItemCrafting ItemLevelUpgradeCrafting(byte craftingNumber, byte targetLevel)
     {
         var crafting = this.Context.CreateNew<ItemCrafting>();
         crafting.Name = $"+{targetLevel} Item Combination";
         crafting.Number = craftingNumber;
         var craftingSettings = this.Context.CreateNew<SimpleCraftingSettings>();
 
         crafting.SimpleCraftingSettings = craftingSettings;
-        craftingSettings.Money = money;
+        craftingSettings.Money = 2_000_000 * (targetLevel - 9);
         craftingSettings.SuccessPercent = (byte)(60 - ((targetLevel - 10) / 2 * 5));
+        craftingSettings.SuccessPercentageAdditionForLuck = 25;
         craftingSettings.SuccessPercentageAdditionForExcellentItem = -10;
-        craftingSettings.SuccessPercentageAdditionForAncientItem = -15;
+        craftingSettings.SuccessPercentageAdditionForAncientItem = -10;
+        craftingSettings.SuccessPercentageAdditionForGuardianItem = -10;
         craftingSettings.SuccessPercentageAdditionForSocketItem = -20;
```

Comment:
Item level upgrade mixes have changed throughout the versions. Newer versions (maybe after S8) have different values that these. Also interesting that several server sources have a cap at 75%, which the client goes over in the case of regular items with luck (85% for +10 mix). Hard to know what was a bug or not. Anyway, now everything matches what the client says.
![Screenshot from 2024-11-15 11-54-02](https://github.com/user-attachments/assets/c00269cf-2989-4a81-88c8-5fa17f414534) ![Screenshot from 2024-11-15 11-49-22](https://github.com/user-attachments/assets/3d11ebf1-b3de-4721-8b90-dfed06da03e0) ![Screenshot from 2024-11-15 12-08-08](https://github.com/user-attachments/assets/2d061ed7-8553-46d5-87bc-22fbb7e538bd)